src/test/debuginfo/repr-u128.rs

```diff
@@ -0,0 +1,22 @@
+// ignore-windows
+//min-system-llvm-version 8.0
+
+//compile-flags: -g -C no-prepopulate-passes
+
+// === GDB TESTS ===================================================================================
+
+// gdb-command: run
```

Comment:
Okay, here is where we need someone who knows debuginfo (@michaelwoerister?).
Or you could peek at a different `debuginfo` test, to see how it makes the debugger print variables automatically and checks the output.